R/ChromBackend.R

```diff
@@ -126,19 +126,20 @@
 #'   supposed to be called right after creating an instance of the
 #'   backend class and should prepare the backend.
 #'   Parameters can be defined freely for each backend, depending on what is
-#'   needed to initialize the backend. It is however suggested to also support
-#'   a parameter `data` that can be used to submit the full data as a
-#'   `DataFrame` to the backend. This would allow the backend to be also
-#'   usable for the [setBackend()] function from `Chromatograms`.
+#'   needed to initialize the backend.
 #'   This method has to ensure to set the spectra variable `dataStorage`
 #'   correctly.
 #'
 #' - `chromData()`, `chromData<-`: gets or sets general chromatogram metadata
-#'   (annotation). `chromData()` returns a `DataFrame`, `chromData<-` expects
-#'   a `DataFrame` with the same number of rows as there are chromatograms in
+#'   (annotation). `chromData()` returns a `data.frame`, `chromData<-` expects
+#'   a `data.frame` with the same number of rows as there are chromatograms in
 #'   `object`. Read-only backends might not need to implement the
 #'   replacement method `chromData<-` (unless some internal caching mechanism
-#'   could be used).
+#'   could be used). `chromData()` should be implemented with the parameter
+#'   `drop` set to `FALSE` as default. this mean that by default the method
```

Comment:
Please reformulate here *... as default. this mean...* - maybe with * ... as default. With `drop = FALSE` the method should return ...*.